nautobot/docs/additional-features/jobhooks.md

```diff
@@ -0,0 +1,21 @@
+# Jobhooks
+
+A jobhook is a mechanism for automatically starting a job when an object is changed. Jobhooks are similar to [webhooks](../models/extras/webhook.md) except change events initiate a `JobHookReceiver` job instead of a web request. Jobhooks are configured in the web UI under Jobs > Jobhooks.
```

Comment:
```suggestion
A jobhook is a mechanism for automatically starting a [job](./jobs.md) when an object is changed. Jobhooks are similar to [webhooks](../models/extras/webhook.md) except change events initiate a `JobHookReceiver` job instead of a web request. Jobhooks are configured in the web UI under Jobs > Jobhooks.
```